handler.go

```diff
@@ -0,0 +1,456 @@
+
+	if err := htmlTemplate.Execute(w, aggregateHealthcheckParams); err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		errorLogger.Printf("Cannot apply params to html template, error was: %v", err.Error())
+		w.Write([]byte("Couldn't render template file for html response"))
+		return
+	}
+}
+
+func buildPodsCheckHtmlResponse(w http.ResponseWriter, healthResult fthealth.HealthResult, environment string, serviceName string) {
+	w.Header().Add("Content-Type", "text/html")
+	htmlTemplate := parseHtmlTemplate(w, healthcheckTemplateName)
+	if htmlTemplate == nil {
+		return
+	}
+
+	aggregateHealthcheckParams := populateAggregatePodChecks(healthResult, environment, serviceName)
+
+	if err := htmlTemplate.Execute(w, aggregateHealthcheckParams); err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		errorLogger.Printf("Cannot apply params to html template, error was: %v", err.Error())
+		w.Write([]byte("Couldn't render template file for html response"))
+		return
+	}
+}
+
+func parseHtmlTemplate(w http.ResponseWriter, templateName string) *template.Template {
+	htmlTemplate, err := template.ParseFiles(templateName)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte("Couldn't open template file for html response"))
+		errorLogger.Printf("Could not parse html template with name %s, error was: %v", templateName, err.Error())
+
+		return nil
+	}
+
+	return htmlTemplate
+}
+
+func populateAggregateServiceChecks(healthResult fthealth.HealthResult, environment string, categories string) *AggregateHealthcheckParams {
+	indiviualServiceChecks, ackCount := populateIndividualServiceChecks(healthResult.Checks)
+	aggregateChecks := &AggregateHealthcheckParams{
+		PageTitle: buildPageTitle(environment, categories),
+		GeneralStatus: getGeneralStatus(healthResult),
+		RefreshFromCachePath: buildRefreshFromCachePath(categories),
+		RefreshWithoutCachePath: buildRefreshWithoutCachePath(categories),
+		AckCount:ackCount,
+		IndividualHealthChecks: indiviualServiceChecks,
+	}
+
+	return aggregateChecks
+}
+
+func buildRefreshFromCachePath(categories string) string {
+	if categories != "" {
+		return fmt.Sprintf("%s?categories=%s", healthcheckPath, categories)
+	}
+
+	return healthcheckPath
+}
+
+func buildRefreshWithoutCachePath(categories string) string {
+	refreshWithoutCachePath := fmt.Sprintf("%s?cache=false", healthcheckPath)
+	if categories != "" {
+		return fmt.Sprintf("%s&categories=%s", refreshWithoutCachePath, categories)
+	}
+
+	return refreshWithoutCachePath
+}
+
+func populateIndividualServiceChecks(checks []fthealth.CheckResult) ([]IndividualHealthcheckParams, int) {
+	var indiviualServiceChecks []IndividualHealthcheckParams
+	ackCount := 0
+	for _, individualCheck := range checks {
+		if individualCheck.Ack != "" {
+			ackCount++
+		}
+
+		hc := IndividualHealthcheckParams{
+			Name: individualCheck.Name,
+			Status: getServiceStatusFromCheck(individualCheck),
+			AvailablePods: "3/3",
```

Comment:
I removed it. Now we are using the "Output" section to display a message if there are unavailable pods, instead of using the 'AvaiulablePods' section